cfme/tests/infrastructure/test_vm_reconfigure.py

```diff
@@ -437,127 +497,62 @@ def test_reconfigure_vm_vmware_mem_multiple():
             4. Cold Decrease
-            5. Hot + Cold Increase
-            6. Hot + Cold Decrease
         expectedResults:
             1. Action should succeed
             2. Action should fail
             3. Action should succeed
             4. Action should succeed
-            5. Action should succeed
-            6. Action should Error
-    """
-    pass
-
-
-@pytest.mark.manual
-@pytest.mark.tier(2)
-@pytest.mark.provider([VMwareProvider])
-def test_reconfigure_vm_vmware_sockets_multiple():
-    """ Test changing the cpu sockets of multiple vms at the same time.
-
-    Polarion:
-        assignee: nansari
-        initialEstimate: 1/6h
-        testtype: functional
-        startsin: 5.6
-        casecomponent: Infra
-        tags: reconfigure
-        testSteps:
-            1. get new configured appliance ->add vmware provider
-            2. provision 2 new vms
-            3. power off 1 vm -> select both vms
-            4. configure-->reconfigure vm
-            5. increase/decrease counts
-            6. power on vm
-            7. check changes
-    """
-    pass
-
-
-@pytest.mark.manual
-@pytest.mark.tier(2)
-@pytest.mark.provider([VMwareProvider])
-def test_reconfigure_vm_vmware_cores_multiple():
-    """ Test changing the cpu cores of multiple vms at the same time.
-
-    Polarion:
-        assignee: nansari
-        initialEstimate: 1/6h
-        testtype: functional
-        startsin: 5.6
-        casecomponent: Infra
-        tags: reconfigure
-        setup:
-            1. get new configured appliance ->add vmware provider
-            2. provision 2 new vms
-        testSteps:
-            1. Hot increase
-            2. Hot Decrease
-            3. Cold Increase
-            4. Cold Decrease
-            5. Hot + Cold Increase
-            6. Hot + Cold Decrease
-        expectedResults:
-            1. Action should fail
-            2. Action should fail
-            3. Action should succeed
-            4. Action should succeed
-            5. Action should fail
-            6. Action should Error
-    """
-    pass
-
-
-@pytest.mark.manual
-@pytest.mark.tier(3)
-@pytest.mark.provider([VMwareProvider])
-def test_reconfigure_add_disk_cold():
-    """ Test adding 16th disk to test how a new scsi controller is handled.
-
-    Bugzilla:
-        1337310
-
-    Polarion:
-        assignee: nansari
-        initialEstimate: 1/6h
-        testtype: functional
-        startsin: 5.7
-        casecomponent: Infra
-        tags: reconfigure
-        testSteps:
-            1. get new configured appliance ->add vmware provider
-            2. provision a new vm with 15 disks
-            3. Add a new disk with CloudForms using the VM Reconfigure dialog
-            4. Check new SCSI controller in vm
-    """
-    pass
-
-
-@pytest.mark.manual
-@pytest.mark.tier(2)
-@pytest.mark.provider([VMwareProvider])
-def test_reconfigure_add_disk_cold_controller_sas():
     """
+    vms = create_vms_modscope
```

Comment:
Optional:

I think you are expecting only one vm to get created here, but I am not sure. I like to do this:
```
vms, = create_vms_modscope
```

This makes clear the only single vm is expected. Also this statement has an advantage that it fails when there are more or less than expected vms returned by the fixture and I think it can be statically checked as well in many situations, so we may see a problem with static type checker.

Reply:
I am expecting 2 vms here. We already have tests to check reconfiguration for 1 VM, we're testing for multiple VMs here.
But to make sure I receive 2 vms, I added one more item({"num_vms": 2}) to the `create_vms_modscope` parametrization to ensure it creates 2 vms, incase someone changes default value in the fixture. So as long as there's no other change in the fixture, it will work just fine for me.